certbot/interfaces.py

```diff
@@ -623,17 +623,19 @@ class GenericUpdater(object):
     """
 
     @abc.abstractmethod
-    def generic_updates(self, domain, *args, **kwargs):
+    def generic_updates(self, lineage, *args, **kwargs):
         """Perform any update types defined by the installer.
 
         If an installer is a subclass of the class containing this method, this
         function will always be called when "certbot renew" is run. If the
         update defined by the installer should be run conditionally, the
         installer needs to handle checking the conditions itself.
 
-        This method is called once for each domain.
+        This method is called once for each lineage.
 
-        :param str domain: domain to handle the updates for
+        :param lineage: Certificate lineage object that is set if certificate
+            was renewed on this run.
```

Comment:
Is this documentation correct? Isn't the `lineage` always set?

Reply:
It's not! Nice catch, same stands true for both of the interfaces.